lib/router/src/lib.rs

```diff
@@ -149,34 +217,19 @@ where
             None => return ResponseFuture::not_recognized(),
         };
 
-        let cache = &mut *self.inner.cache.lock().expect("lock router cache");
-
-        // First, try to load a cached route for `target`.
-        if let Some(service) = cache.access(&target) {
-            return ResponseFuture::new(request, service.clone());
-        }
-
-        // Since there wasn't a cached route, ensure that there is capacity for a
-        // new one.
-        let reserve = match cache.reserve() {
-            Ok(r) => r,
-            Err(cache::CapacityExhausted { capacity }) => {
-                return ResponseFuture::no_capacity(capacity);
-            }
-        };
-
-        // Bind a new route, send the request on the route, and cache the route.
-        let service = LoadShed::new(self.inner.make.make(&target));
-
-        reserve.store(target, service.clone());
-        ResponseFuture::new(request, service)
+        return ResponseFuture::new(
```

Comment:
I think the return here is not necessary.
```suggestion
       ResponseFuture::new(
```